go/cmd/dolt/dolt.go

```diff
@@ -146,6 +148,10 @@ func runMain() int {
 
 	dEnv := env.Load(context.TODO(), env.GetCurrentUserHomeDir, filesys.LocalFS, doltdb.LocalDirDoltDB, Version)
 
+	if dEnv.DBLoadError == nil && commandWillMigrate(args) {
```

Comment:
I agree that we should have a migration command. 
I left the migration as automated so that we can use the compatibility [CI tests](https://jenkins.awsdev.ld-corp.com/job/liquidata-inc/job/dolt/job/PR-491/). 
I can add the migration to this PR or another PR before the next release.